src/features/author.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-param-reassign */
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+
+type State = {
+  author: User | null
+};
+
+const initialState: State | null = {
```

Comment:
The initialState is declared as State or null, but it is always initialized with a State object. Remove the null type from the initialState declaration.